lib/tasks/update-pension-calc-tool-name.rake

```diff
@@ -0,0 +1,18 @@
+namespace :corporate_partners do
+  desc 'Update tool name from "Pensions calculator" to "Pension calculator"'
+  task update_tool_name: :environment do
+    records_to_update = CorporatePartner.all.select do |partner|
+      partner.tool_name == 'Pensions calculator'
+    end
+
+    CorporatePartner.transaction do
+      records_to_update.each do |record|
```

Comment:
you can avoid doing 5x DB updates with a `records_to_update.update_all(tool_name: 'Pension Calculator')`, that will execute a single DB update.